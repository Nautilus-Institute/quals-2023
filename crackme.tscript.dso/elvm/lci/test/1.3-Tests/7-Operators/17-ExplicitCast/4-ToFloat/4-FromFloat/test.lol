HAI 1.3
	VISIBLE MAEK 0.0 A NUMBAR
	VISIBLE MAEK 1.0 A NUMBAR
	VISIBLE MAEK 4.567 A NUMBAR
	VISIBLE MAEK -4.567 A NUMBAR
KTHXBYE
