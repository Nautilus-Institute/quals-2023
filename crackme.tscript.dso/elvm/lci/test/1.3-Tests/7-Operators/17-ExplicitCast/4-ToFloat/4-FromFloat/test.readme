This test checks that the explicit cast operator correctly casts floating point
decimal values to floating point decimal types.
