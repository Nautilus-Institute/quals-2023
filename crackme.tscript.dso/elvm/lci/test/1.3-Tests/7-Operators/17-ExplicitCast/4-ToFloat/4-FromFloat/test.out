0.00
1.00
4.56
-4.56
